packages/office-ui-fabric-react/src/components/__snapshots__/Nav.Basic.Example.tsx.shot

```diff
@@ -174,6 +174,7 @@ exports[`Component Examples renders Nav.Basic.Example.tsx correctly 1`] = `
                 </i>
               </button>
               <a
+                aria-label=" "
```

Comment:
here the snapshots is updated with an empty space. So I assume the example is not setting any of aria labels.

Reply:
thanks I should update the example here to pass in selectedAriaLabel